plugin-server/src/cdp/services/messaging/mailjet-manager.service.test.ts

```diff
@@ -0,0 +1,256 @@
+import crypto from 'crypto'
+import express from 'express'
+
+import { Hub } from '../../../types'
+import { parseJSON } from '../../../utils/json-parse'
+import { MessagingMailjetManagerService } from './mailjet-manager.service'
+
+describe('MessagingMailjetManagerService', () => {
```

Comment:
I'm picky about tests because tests are important but also a source of pain over time if they aren't well maintained. A few things that could be done way better here commented where they are